worker/src/RTC/SeqManager.cpp

```diff
@@ -77,28 +77,30 @@ namespace RTC
 		if (!this->dropped.empty())
 		{
 			// Delete dropped inputs older than input - MaxValue/2.
-			size_t droppedSize = this->dropped.size();
-			auto it            = this->dropped.lower_bound(input - MaxValue / 2);
+			size_t droppedCount = this->dropped.size();
+			auto it             = this->dropped.lower_bound(input - MaxValue / 2);
 
 			this->dropped.erase(this->dropped.begin(), it);
```

Comment:
Here it no longer removes entries from `this->dropped`. Is this ok?

Reply:
The code you quoted is the same as before, only rename `droppedSize` to `droppedCount`
We first remove items lower than (input-MaxValue/2) and recalculate this->base. 
Then count the items lower than input and use this->base minus it.

Reply:
This commit works the same as before, only replace`find` & `upper_bound` with `lower_bound`.
Because when input is in dropped set, `lower_bound` will return as 'find', otherwise, it will return as `upper_bound`.

Reply:
sorry, my fault.